chpl/chpl-resources/src/main/resources/errors.properties

```
@@ -402,6 +402,14 @@ surveillance.errorAdding=Error adding a new pending surveillance. Please make su
 surveillance.permissionError=User %s does not have access to add surveillance.
 surveillance.permissionErrorWithAcb=User %s does not have access to add surveillance to %s as the listing is owned by ONC-ACB %s.
 
+# surveillance reports
+report.quarterlySurveillance.missingYear=A year must be specified for the quarterly report.
+report.quarterlySurveillance.missingAcb=An ACB must be specified for the quarterly report.
```

```suggestion
report.quarterlySurveillance.missingAcb=An ONC-ACB must be specified for the quarterly report.
```

Public facing text should always have the "ONC-" part in it